multinode/nodes/service.go

```diff
@@ -315,6 +318,10 @@ func (service *Service) TrustedSatellites(ctx context.Context) (_ storj.NodeURLs
 
 	var trustedSatellites storj.NodeURLs
 	for _, node := range listNodes {
+		nodeinfo := service.FetchNodeInfo(ctx, node)
```

Comment:
We would also use here the function that checks the status of the node

Reply:
@ifraixedes the nodestatus function only checks where the node is `online` or `offline` based on it's last contract. But the nodes can have other status also I think I should create a separate function to check the status of the node and use it every where.

Reply:
> I think I should create a separate function to check the status of the node and use it every where

That's what I wanted to say in my comment https://github.com/storj/storj/pull/7146/files#r1768432738

And use it here and in other places where I dropped the same comment.

Reply:
Oops my bad. I thought you are talking about the nodeStatus function. Sorry. 
I am creating a new one.